src/state/sagas/windows.js

```diff
@@ -54,7 +54,9 @@ export function* setWindowStartingCanvas(action) {
         || miradorManifest.canvasAt(canvasIndex || 0)
         || miradorManifest.canvasAt(0);
       if (startCanvas) {
-        const thunk = yield call(setCanvas, windowId, startCanvas.id);
+        const thunk = yield call(
+          setCanvas, windowId, startCanvas.id, null, { preserveViewport: true },
```

Comment:
I'm not sure about this one; we end up here if the window properties didn't call out a specific canvas